dolphinscheduler-dao/src/main/java/org/apache/dolphinscheduler/dao/mapper/ErrorCommandMapper.java

```diff
@@ -43,4 +45,10 @@ List<CommandCount> countCommandState(
                                          @Param("startTime") Date startTime,
                                          @Param("endTime") Date endTime,
                                          @Param("projectCodes") List<Long> projectCodes);
+
+    IPage<ErrorCommand> queryErrorCommandPage(Page<ErrorCommand> page);
+
+    IPage<ErrorCommand> queryErrorCommandPageByIds(Page<ErrorCommand> page,
+                                                   @Param("definitionCodes") List<Long> definitionCodes,
+                                                   @Param("projectCode") Long projectCode);
```

Comment:
```suggestion
    IPage<ErrorCommand> queryErrorCommandPageByIds(Page<ErrorCommand> page,
                                                   @Param("workflowDefinitionCodes") List<Long> workflowDefinitionCodes);
```
Please remove the projectCode at this method, this param is never used?
And use more accurate naming for `definitionCodes`, since there exist `workflowDefinitionCodes` and `taskDefinitionCodes`